src/migrations/20240807172716-reset-session-completions.js

```diff
@@ -0,0 +1,36 @@
+const {
+  prepMigration,
+} = require('../lib/migration');
+
+module.exports = {
+  up: async (queryInterface) => queryInterface.sequelize.transaction(
+    async (transaction) => {
+      await prepMigration(queryInterface, transaction, __filename);
+      await queryInterface.sequelize.query(/* sql */`
+        -- First, set it at the event level.
```

Comment:
what is "it"

Reply:
These from the description:

* `pocComplete` to `false`
* `pocCompleteId` to `""`
* `pocCompleteDate` to `""`

I can put this comment in the migration too, though